src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -193,6 +194,7 @@ private static IList<Exception> Flatten(ICollection<Exception> exceptions)
             var aggregate = new AggregateException(exceptions);
             return aggregate.Flatten().InnerExceptions
                 .Select(UnwrapException)
+                .Where(ex => !(ex is JoinableTaskContextException))
```

Comment:
@jmarolf You can just disable the outlining feature of the editor as I've done in a different PR. Our outlining unit tests directly test our outlining implementation without going through the editor.

https://github.com/dotnet/roslyn/pull/20397/files#diff-6159a3d84595fb60c93e257f29db681d